### ISSUE TITLE
Visual Studio extension template project for .NET core/6.0.

### ISSUE THREAD

**Issue by PanchRajMaurya** [VsixCommunity/Community.VisualStudio.Toolkit#446]
All the visual studio extensions are in .NET Framework. I have some assemblies developed in .net6.0 that need to be used in Visual Studio extension. Can I create Visual Studio extension in .net6.0? or any other alternative to achieve my requirements?

**Comment by ErikEJ** [comment]
No, you can not. I simply communicate with .NET 6 code via a console app and json messages.

**Comment by PanchRajMaurya** [comment]
Thanks for your quick reply. Can you provide some code sample/sample project to open  WinForms  that exists in an assembly developed in .NET6.0 as a command in Visual Studio extension?
